From e806493834a1e890b93fdc654cdb4e42b4630f02 Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Tue, 23 Mar 2010 21:25:39 +0000
Subject: [PATCH 029/135] dsp-bridge: deh: fix obvious return codes

Now that the code is cleaner is easy to spot them.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
---
 drivers/dsp/bridge/wmd/ue_deh.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/dsp/bridge/wmd/ue_deh.c b/drivers/dsp/bridge/wmd/ue_deh.c
index 10cc883..657e139 100644
--- a/drivers/dsp/bridge/wmd/ue_deh.c
+++ b/drivers/dsp/bridge/wmd/ue_deh.c
@@ -138,7 +138,7 @@ dsp_status bridge_deh_destroy(struct deh_mgr *hdeh_mgr)
 	struct deh_mgr *deh_mgr_obj = (struct deh_mgr *)hdeh_mgr;
 
 	if (!MEM_IS_VALID_HANDLE(deh_mgr_obj, SIGNATURE))
-		return DSP_SOK;
+		return DSP_EHANDLE;
 
 	/* Release dummy VA buffer */
 	bridge_deh_release_dummy_mem();
-- 
1.7.0.4

